ehr2meds/PREMEDS/azure_run/model.py

```
@@ -47,72 +56,78 @@ def load(model_name, version=None, meta_only=False):
-        model._path       = path
+        model._path = path
 
         if meta_only:
             model._meta_only = True
         else:
             model._load(path)
-        
+
         return model
 
     def register(self, name=None, exist_ok=False, path=None, datasets=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)